src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerViewController.swift

```diff
@@ -72,11 +78,10 @@ class QRScannerViewController: UIViewController {
 
 	private let focusView = QRScannerFocusView()
 	private let dismiss: () -> Void
-
+	private let flashButtonTag = 12
```

Comment:
Maybe nothing for now, but we have a Tag on a view to find it later on? 😳🙀

Reply:
I'm just as surprised as you are 😆

Reply:
Already on it.

Reply:
I just was in a call with @naveeddotio  add asked him to change that.

Reply:
I just took the legacy wrote already there written by some legend :D